packages/flutter_tools/lib/src/base/file_system.dart

```diff
@@ -17,6 +17,40 @@ import 'signals.dart';
 // the tool is killed by a signal.
 export 'package:file/file.dart';
 
+extension FileUtils on File {
```

Comment:
Does this extension apply to every class that extends or implements `File`, or does it apply *only* to exactly `File`? Either way, it is probably worth a comment here.

Reply:
Moved to ErrorHandlingFileSystem